Homework/css/styles.css

```diff
@@ -0,0 +1,95 @@
+
+body {	
+
+	background: rgb(197,222,234); /* Old browsers */
+	background: -moz-linear-gradient(top, rgba(197,222,234,1) 0%, rgba(138,187,215,1) 31%, rgba(6,109,171,1) 100%); /* FF3.6-15 */
+	background: -webkit-linear-gradient(top, rgba(197,222,234,1) 0%,rgba(138,187,215,1) 31%,rgba(6,109,171,1) 100%); /* Chrome10-25,Safari5.1-6 */
+	background: linear-gradient(to bottom, rgba(197,222,234,1) 0%,rgba(138,187,215,1) 31%,rgba(6,109,171,1) 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+	filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#c5deea', endColorstr='#066dab',GradientType=0 ); /* IE6-9 */
+	
+	font-family: sans-serif;
+	margin-right: 200px;
+	margin-left: 200px;
+
+}
+
+
+
+h1 {
+	color:#1C4D1D;
+	text-align: center;
+	font-family: monospace;
+	background: #1C4D1D;
+	padding: 15px;
+	border: 3px solid black;
+}
+
+h3 {
+	color:#1C4D1D;
+}
+
+p ul {
+	color:#1C4D1D;
+	text-align: center;
+}
+
+nav {
+	text-align: center;
+	color: red;
```

Comment:
:hammer:  so it looks like this color isn't actually styling the nav links red. Perhaps we needed to target those links a bit more specifically? Assuming this was your intentions, of course...